### ISSUE TITLE
Malaria Detection CNN

### ISSUE THREAD

**Issue by willjhay** [willjhay/DS-Projects#6]
CNN classifier using TensorFlow, 2DConv, ReLu, Max Pooling, Adam optimizer, Batch Normalization
